chapter_linear-networks/linear-regression_vn.md

```diff
@@ -391,7 +394,10 @@ Note that even if our function is truly linear and noiseless, these parameters w
 although the algorithm converges slowly towards a local minimum it cannot achieve it exactly in a finite number of steps.
 -->
 
-*dịch đoạn phía trên*
+Sau khi huấn luyện đủ số vòng lặp được xác định trước (hoặc đạt được một tiêu chí dừng nhất định khác),
+ta sẽ ghi lại các tham số mô hình đã được ước lượng, ký hiệu là $\hat{\mathbf{w}}, \hat{b}$ (thường thì biểu tượng "mũ" thể hiện các giá trị ước lượng).
+Lưu ý rằng ngay cả khi hàm số thực sự tuyến tính và không có nhiễu, các tham số này sẽ không tối thiểu hóa được hàm mất mát.
+Mặc dù thuật toán dần dần hội tụ đến một cực tiểu địa phương, nó vẫn không thể tới được chính xác cực tiểu đó với số bước hữu hạn.
```

Comment:
```suggestion
Mặc dù thuật toán dần dần hội tụ đến một cực tiểu cục bộ, nó vẫn không thể tới được chính xác cực tiểu đó với số bước hữu hạn.
```

Reply:
Mình google lại thấy `địa phương` được dùng phổ biến hơn

Reply:
@duythanhvn ới

Reply:
Phần này mình không có nguồn nào cụ thể chỉ ra nó nên là `địa phương` hay là `cục bộ`. Cả hai cách dịch này mình đều thấy dùng, ở góc độ của mình thì mình nghe và thường dùng qua từ `cục bộ`.

Tựu chung thì nó ở mức độ cá nhân thôi, nghe `cục bộ` nó cũng kỹ thuật hơn so với địa phương.

Reply:
hmm có lý, mình cũng chưa tiếp xúc với toán nhiều lắm nên thống nhất theo bạn vậy